src/main/resources/application.properties

```diff
@@ -4,3 +4,5 @@ spring.datasource.password= postgres2020.
 spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.PostgreSQLDialect
```

Comment:
This shouldn't be here

Reply:
Please review again, this branch wasn't update with main.